### PR TITLE
Add Avalanche Aave v2 events

### PR DESCRIPTION
**Pull request by marcinja**
## What?
ie: Add support for uniswap contract events 

## How? 
ie: I used [abi-parser](https://nansen-contract-parser-prod.web.app/) or cli tools to build the table defenitions

## Related PRs (optional)
PRs that are related to this or may need to be deployed before this PR

### CODE REVIEW

File: parse/table_definitions_avalanche/aave/AToken_v2_event_Approval.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "value",
+                    "type": "uint256"
+                }
+            ],
+            "name": "Approval",
+            "type": "event"
+        },
+        "contract_address": "SELECT aToken FROM ref('LendingPoolConfigurator_v2_event_ReserveInitialized')",
```
Comment (daishuai323) [inline parse/table_definitions_avalanche/aave/AToken_v2_event_Approval.json:28]: Hi for this kind of reference, the referred event (here `LendingPoolConfigurator_v2_event_ReserveInitialized`) needs to be submitted through a separate PR and merged first, then we can merge the current PR, so the tables can be built in order. Can you pls update thanks!

Comment (marcinja) [reply parse/table_definitions_avalanche/aave/AToken_v2_event_Approval.json:28]: @daishuai323 OK sounds good. I made the PR for LendingPoolConfigurator_v2 events here: https://github.com/nansen-ai/evmchain-etl-table-definitions/pull/119
